src/main/java/io/jenkins/plugins/checks/status/BuildStatusChecksPublisher.java

```diff
@@ -218,6 +222,7 @@ public void onNewHead(final FlowNode node) {
                 run = (Run) node.getExecution().getOwner().getExecutable();
             }
             catch (IOException e) {
+                LOGGER.log(Level.WARNING, "Unable to find Run from flow node.");
```

Comment:
log the exception?

should work I think but java.util.logging api is rubbish
```suggestion
                LOGGER.log(Level.WARNING, "Unable to find Run from flow node.", e);
```